config/foreman.yaml

```diff
@@ -3,14 +3,14 @@
 
 ## Installer configuration
 # Path to answer file
-:answer_file: ./config/answers.yaml
+:answer_file: ./config/foreman-answers.yaml
 :installer_dir: .
 # Uncomment if you want to load puppet modules from a specific path, $pwd/modules is used by default
-:modules_dir: ./_build/modules
+:module_dirs: ./_build/modules
```

Comment:
I know this is here from before, but if this is reduced to './modules', then for running from git I can easily do:

git clone https://github.com/theforeman/foreman-installer.git
gem install kafo or RPM install
cd foreman-installer
./bin/foreman-installer -v